docs/content/latest/releases/releases-overview.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Supported and planned releases
+headerTitle: Supported and planned releases
+linkTitle: Supported and planned releases
+description: An overview of the latest and current stable releases, and planned releases.
+section: RELEASES
+menu:
+  latest:
+    identifier: releases-overview
+    weight: 2588 
+isTocNested: true
+showAsideToc: true
+---
+
+{{< note title="Important" >}}
+
+As of version `2.2`, Yugabyte is following a new release versioning convention for YugabyteDB releases.  For details, see [Release versioning](../versioning).
+
+{{< /note >}}
+
+## Supported stable releases
```

Comment:
My preference is to combine this table and the one in section below. Just add a column that outlines the recommended latest and stable release. That way, there is one table. This fragmentation is confusing.

Also, a couple of lines below that table, helping the end user quickly decide which release they should pick would be great.

Reply:
@rkarthik007 do we want to advertise EOL & maintenance support dates for the latest release? I would imagine NO. 
I think adding the latest and future scheduled releases in the same table as supported stable releases are confusing. Can we keep the way it is for now and roll with it?

Reply:
The stable and latest releases releases are combined into one section and table.